src/components/Resources/Category/CustomCard.js

```diff
@@ -49,7 +49,7 @@ export default function CustomCard({
           </Card>
         </Link>
       </CardActionArea>
-      <DeleteCategoryButton categoryFirestoreId={id} />
+      {id !== 'new' && <DeleteCategoryButton categoryFirestoreId={id} />}
```

Comment:
Instead of using the id field to indicate whether a card is the add category component, create a boolean hook, and pass the knowledge of if the card is the add category component via props.